js/ui/easings.js

```diff
@@ -194,7 +194,7 @@ util.extend(exports, {
             this.flyTo(center, zoom, 0, options);
     },
 
-    easeTo: function(latlng, zoom, bearing, options) {
+    easeTo: function(latlng, zoom, bearing, pitch, options) {
```

Comment:
I think so. Especially if we want to expose altitude too.

Or maybe separating the view from the ease options make sense: `easeTo(view, options)`?